src/Nokia_LCD.cpp

```diff
@@ -63,6 +76,14 @@ void Nokia_LCD::setInverted(bool invert)
     inverted = invert;
 }
 
+void Nokia_LCD::setBacklight(bool enabled)
+{
+    if (!kBl_pin) {
+        return;
+    }
+    digitalWrite(kBl_pin, enabled);
+}
```

Comment:
To begin with, you have forgotten to initialize `kBl_pin` as an `OUTPUT` in the constructor. Aside of that, I have mixed feelings about the method.
First, we need to find a default value for the pin that will always work. I assume creating a constant with a really high value (e.g. `255`) will be fine since it's unlikely the value will be used for an actual/valid pin.

Second, API design speaking, would you say that setting or clearing the backlight should be a concern of the class? If you think yes, then I am fine with it. Otherwise, it can be the concern of "the user" of the class and therefore `setBacklight` should not be a member function.

Reply:
> First, we need to find a default value for the pin that will always work. I assume creating a constant with a really high value (e.g. 255) will be fine since it's unlikely the value will be used for an actual/valid pin.

Alternatively, we can go with the *two* constructors and set a dedicated boolean `usingBacklight` which determines whether the `setBacklight` method should do something or not.

Reply:
Hi!
I have added the new class member usingBacklight
And now I am initing the backlight pin as 255.
Also, setting the right pinMode at the begin method.

About the design. I believe, that the lib should control all aspects of a device, including it's backlight capability. Also, from my user classes, I don't want to implement a login that, as I expect, should be done in the lib.